src/stk/molecular/key_makers/utilities.py

```diff
@@ -37,3 +40,22 @@ def get_inchi_key(molecule):
     """
 
     return rdkit.MolToInchiKey(molecule.to_rdkit_mol())
+
+
+def get_smiles(molecule):
+    """
+    Get the RDKit canonical, isomeric SMILES of `molecule`.
+
+    Parameters
+    ----------
+    molecule : :class:`.Molecule`
+        The molecule whose SMILES is required.
+
+    Returns
+    -------
+    :class:`str`
+        The SMILES.
+
+    """
+    rdkit_mol = rdkit.RemoveHs(molecule.to_rdkit_mol())
+    return rdkit.MolToSmiles(rdkit_mol)
```

Comment:
I know they are on by default, but it might be a good idea to explicitly set the isomeric and canoncical smiles on, in case there is an api change in rdkit.